conda/core/link.py

```diff
@@ -515,49 +537,93 @@ def _verify(cls, prefix_setups, prefix_action_groups):
         )
         if transaction_exceptions:
             return transaction_exceptions
-        exceptions = tuple(exc for exc in concatv(
-            concat(cls._verify_individual_level(prefix_group)
-                   for prefix_group in itervalues(prefix_action_groups)),
-            concat(cls._verify_prefix_level(target_prefix, prefix_group)
-                   for target_prefix, prefix_group in iteritems(prefix_action_groups)),
-        ) if exc)
+
+        exceptions = []
+        with ThreadLimitedThreadPoolExecutor() as executor:
+            futures = list(executor.submit(cls._verify_individual_level, pg)
+                           for pg in itervalues(prefix_action_groups))
+            futures.extend(executor.submit(cls._verify_prefix_level, target_prefix, pg)
+                           for target_prefix, pg in iteritems(prefix_action_groups))
+            for future in as_completed(futures):
+                if future.result():
+                    exceptions.extend(future.result())
         return exceptions
 
     @classmethod
     def _execute(cls, all_action_groups):
         # unlink unlink_action_groups and unregister_action_groups
-        unlink_actions = tuple(
-            group for group in all_action_groups if group.type in ("unlink", "unregister"))
+        unlink_actions = tuple(group for group in all_action_groups if group.type == "unlink")
```

Comment:
what happened to the unregister group type?

Reply:
These come in a little later: https://github.com/conda/conda/pull/8670/files#diff-315c016a32ef55653e67795ff2b18b7cR613